invokeai/backend/ip_adapter/attention_processor.py

```diff
@@ -127,26 +133,35 @@ def __call__(
             for ipa_embed, ipa_weights, scale in zip(ip_adapter_image_prompt_embeds, self._weights, self._scales):
                 # The batch dimensions should match.
                 assert ipa_embed.shape[0] == encoder_hidden_states.shape[0]
-                # The channel dimensions should match.
-                assert ipa_embed.shape[2] == encoder_hidden_states.shape[2]
+                # The token_len dimensions should match.
+                assert ipa_embed.shape[-1] == encoder_hidden_states.shape[-1]
 
                 ip_hidden_states = ipa_embed
 
+                # ip_hidden_state.shape = (batch_size, num_ip_images, ip_seq_len, ip_image_embedding)
+
```

Comment:
is this just showing the expected shape or is this something that should be uncommented in the future?

Reply:
This is just the expected shape. I updated the comments in [3079c75](https://github.com/invoke-ai/InvokeAI/pull/4882/commits/3079c75a60c01f5478f3230a8f74159d347fb1a0) to make this more clear.